tooling/sync/lib/command_surface_loader.py

```
@@ -0,0 +1,237 @@
+        resolved.relative_to(root)
+    except ValueError as exc:
+        raise CommandSurfaceLoadError("manifest_path must stay within repo_root") from exc
+    return resolved
+
+
+def _normalize_manifest_ref(repo_root: Path, resolved_path: Path) -> str:
+    root = repo_root.resolve()
+    try:
+        return str(resolved_path.resolve().relative_to(root))
+    except ValueError as exc:
+        raise CommandSurfaceLoadError("manifest_path must stay within repo_root") from exc
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)